### PR TITLE
RCP improvements for 1.1

### PR DESCRIPTION
**Pull request by emizan76**
- Renamed dir 1.1.0 to training_1.1.0
- Removed hparams that should always be the same as reference
- Added a couple of missing hparams
- Added "creator" field (who provided RCP)
- Added "when" field (for which round and prior/at submission)
- Added "platform" field with details on HW/SW stack used to generate RCP.

Still some fixes to be made.
Refer to this spreadsheet for RCPs: https://docs.google.com/spreadsheets/d/1pVQrfYNNI2n3J2LSZT875INohWMfCMziP65Gc85TR7s/edit#gid=353265666

### CODE REVIEW

File: mlperf_logging/rcp_checker/rcp_checker.py [hunk 1]
```diff
@@ -83,8 +83,8 @@ def get_submission_epochs(result_files, benchmark, bert_train_samples):
 class RCP_Checker:
 
     def __init__(self, usage, ruleset, verbose, bert_train_samples):
-        if ruleset != '1.0.0':
-            raise Exception('RCP Checker only supported in 1.0.0')
+        if ruleset != '1.0.0'  and ruleset != "1.1.0":
```
Comment (shangw-nvidia) [inline mlperf_logging/rcp_checker/rcp_checker.py:86]:
```python
if ruleset not in {'1.0.0', '1.1.0'}:
```
might be cleaner,

File: mlperf_logging/rcp_checker/training_1.1.0/rcps_bert.json [hunk 2]
```diff
@@ -143,6 +164,9 @@
   "bert_ref_12288":
   {
     "Benchmark": "bert",
+    "Creator": "NVidia",
```
Comment (shangw-nvidia) [inline mlperf_logging/rcp_checker/training_1.1.0/rcps_bert.json:167]: Actually, technically speaking, it's `NVIDIA` (i.e., all capital letters).

File: mlperf_logging/rcp_checker/training_1.1.0/rcps_dlrm.json [hunk 3]
```diff
@@ -3,13 +3,15 @@
...
+    "Creator": "NVidia",
+    "When": "Prior to 1.0 submission",
+    "Platform": "TBD",
     "BS": 32768,
     "Hyperparams": {
        "opt_base_learning_rate": 18,
        "opt_learning_rate_warmup_steps": 8000,
        "lr_decay_start_steps": 70000,
        "sgd_opt_base_learning_rate": 18,
-       "sgd_opt_learning_rate_decay_poly_power": 2,
```
Comment (shangw-nvidia) [inline mlperf_logging/rcp_checker/training_1.1.0/rcps_dlrm.json:12]: Is removing `sgd_opt_learning_rate_decay_poly_power` discussed in the training WG? I'm unaware of this.

File: mlperf_logging/rcp_checker/training_1.1.0/rcps_maskrcnn.json [hunk 4]
```diff
@@ -3,13 +3,13 @@
   "maskrcnn_ref_48":
   {
     "Benchmark": "maskrcnn",
+    "Creator": "NVidia",
+    "When": "Prior to 1.0 submission",
+    "Platform": "TBD",
     "BS": 48,
     "Hyperparams": {
-      "opt_learning_decay_factor": 0.1,
```
Comment (shangw-nvidia) [inline mlperf_logging/rcp_checker/training_1.1.0/rcps_maskrcnn.json:8]: Is removing `opt_learning_decay_factor`, `max_image_size` and `min_image_size` discussed in the training WG?

File: mlperf_logging/rcp_checker/training_1.1.0/rcps_resnet.json [hunk 5]
```diff
@@ -3,40 +3,40 @@
...
+    "Creator": "NVidia",
+    "When": "At 1.0 submission",
+    "Platform": "TBD",
     "BS": 1632,
     "Hyperparams": {
       "optimizer": "lars",
       "opt_base_learning_rate": 7.4,
-      "opt_end_learning_rate": 1e-4,
-      "opt_learning_rate_decay_poly_power": 2,
-      "epsilon": 0,
```
Comment (shangw-nvidia) [inline mlperf_logging/rcp_checker/training_1.1.0/rcps_resnet.json:12]: Hyper-parameter changes discussed in the training WG?

File: mlperf_logging/rcp_checker/training_1.1.0/rcps_rnnt.json [hunk 6]
```diff
@@ -13,11 +16,11 @@
       "opt_lamb_beta_1": 0.9,
       "opt_lamb_beta_2": 0.999,
       "opt_lamb_learning_rate_min": 1e-5,
-      "opt_gradient_clip_norm": 1.0,
```
Comment (shangw-nvidia) [inline mlperf_logging/rcp_checker/training_1.1.0/rcps_rnnt.json:16]: Hyper-parameter changes discussed in the training WG?

File: mlperf_logging/rcp_checker/training_1.1.0/rcps_ssd.json [hunk 7]
```diff
@@ -3,12 +3,15 @@
   "ssd_ref_1024_s128":
   {
     "Benchmark": "ssd",
+    "Creator": "NVidia",
+    "When": "Prior 1.0 submission",
+    "Platform": "TBD",
     "BS": 1024,
     "Hyperparams": {
-      "opt_base_learning_rate": 2.5e-3,
```
Comment (shangw-nvidia) [inline mlperf_logging/rcp_checker/training_1.1.0/rcps_ssd.json:8]: Hyper-parameter changes discussed in the training WG?

**Review by shangw-nvidia — COMMENTED** [review]
Looks good overall. I saw some hyper-parameters in the RCPs being changed in this PR? I'm not aware of (the discussion related to) these, so I left comments just to double check that they are not changed by mistakes.

File: mlperf_logging/rcp_checker/training_1.1.0/rcps_dlrm.json (hunk 3 above)
Comment (emizan76) [reply mlperf_logging/rcp_checker/training_1.1.0/rcps_dlrm.json:12]: This is tied to reference, so there is no point keep that in the file. The training WG is aware of that.

File: mlperf_logging/rcp_checker/training_1.1.0/rcps_bert.json (hunk 2 above)
Comment (emizan76) [reply mlperf_logging/rcp_checker/training_1.1.0/rcps_bert.json:167]: Ha, ha. Then your name should be shangw-NVIDIA :)

File: mlperf_logging/rcp_checker/rcp_checker.py (hunk 1 above)
Comment (emizan76) [reply mlperf_logging/rcp_checker/rcp_checker.py:86]: Will do

**Review by shangw-nvidia — APPROVED** [review]

**Review by xyhuang — APPROVED** [review]

**Pull request merged**